.github/workflows/ci.yml

```diff
@@ -73,36 +74,47 @@ jobs:
 
             - run: yarn encore production
 
-            - name: Bootstrap database
-              run: |
-                  bin/post-install-dev.sh
-
             - run: composer validate
               if: always()
 
-            - run: php bin/console lint:twig templates
+            - run: APP_DEBUG=1 bin/console c:w
               if: always()
 
-            - run: php bin/console lint:yaml config --parse-tags
+            - run: bin/console lint:twig templates
               if: always()
 
-            - run: php vendor/bin/php-cs-fixer fix --verbose --diff --dry-run --format=checkstyle | cs2pr
+            - run: bin/console lint:yaml config --parse-tags
+              if: always()
+
+            - run: vendor/bin/php-cs-fixer fix --verbose --diff --dry-run --format=checkstyle | cs2pr
               if: always()
 
             - run: vendor/bin/phpcs --warning-severity=0 --report=checkstyle | cs2pr
               working-directory: ${{ matrix.working_directory }}
               if: always()
 
-            - run: php bin/console doctrine:schema:validate
+            - name: Run PHPStan
+              run: vendor/bin/phpstan analyse --configuration phpstan.neon --level max --no-progress . --error-format=checkstyle | cs2pr
               if: always()
 
-            - name: Run PHPStan
-              run: php vendor/bin/phpstan analyse --configuration phpstan.neon --level max --no-progress . --error-format=checkstyle | cs2pr
+            - name: Bootstrap database
+              run: |
```

Comment:
why not use the bin/post-install-dev.sh ?

Reply:
Because we don't need everything in this script (features will be loaded by tests if necessary, and we want the pipeline to fail if there is something wrong with the migrations